_codux/mocks/fakers.ts

```diff
@@ -137,3 +137,14 @@ export function getCartTotals(): CartTotals {
         },
     };
 }
+
+export function createCategory(settings?: FakeDataSettings): Category {
```

Comment:
shouldn't category object contain some products ?

Reply:
no, there is no products property in category